src/library/dlgtrackinfo.cpp

```diff
@@ -501,14 +501,14 @@ void DlgTrackInfo::slotBpmHalve() {
 }
 
 void DlgTrackInfo::slotBpmTwoThirds() {
-    m_pBeatsClone->scale(Beats::Beats::TWOTHIRDS);
+    m_pBeatsClone->scale(Beats::TWOTHIRDS);
```

Comment:
here, I don't know what I did. 
I just made it the same way as the above block with Beats::HALVE
It compiles, but does it "means" the same thing ?